.idea/jarRepositories.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this worth having in VCS? In general, probably best to have the fewest `.idea` files version controlled.

Reply:
According with the Jetbrains Team, it is fine to add all of those files:
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

Reply:
It's fine in the sense that it won't break anything, but having them in the repo is unnecessary. This one, for example, is just generated from our `build.gradle` files which have the repo configs in them.